torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -30,8 +32,8 @@ class Hypothesis(NamedTuple):
     score: float
 
 
-class KenLMLexiconDecoder:
-    """torchaudio.prototype.ctc_decoder.KenLMLexiconDecoder()
+class LexiconDecoder:
+    """torchaudio.prototype.ctc_decoder.LMLexiconDecoder()
```

Comment:
```suggestion
    """torchaudio.prototype.ctc_decoder.LexiconDecoder()
```

and similar for the docstring for `Hypothesis`